mcs/class/System.Private.CoreLib/System/__ComObject.cs

```diff
@@ -0,0 +1,10 @@
+namespace System
+{
+	internal class __ComObject
```

Comment:
Is it really needed?

Reply:
The runtime asserts on it.

Reply:
Can we remove it? Otherwise, we'll have to teach linker etc which might be even more work

Reply:
Its needed for COM support, we can get rid of it, but its more work than adding this one class.